src/SDKs/HDInsight/Management.HDInsight.Tests/Management.HDInsight.Tests.csproj

```diff
@@ -0,0 +1,84 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HDInsight.Tests</PackageId>
+    <Description>HDInsight.Tests Class Library</Description>
+    <AssemblyName>Microsoft.Azure.Management.HDInsight.Tests</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="WindowsAzure.Storage" Version="8.1.4" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Management.HDInsight\Management.HDInsight.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="SessionRecords\Management.HDInsight.Tests.CapabilitiesTests\TestGetCapabilities.json">
```

Comment:
@wawon-msft replace all these entries as below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/BatchAI.Tests/BatchAI.Tests.csproj#L18

Also you need to use 1.6.0 or greater version of Resource manager for recording all your tests. You will need to add 1.6.0 version as a nuget reference and use it. The one that is by default added is 1.1.0 which is really old and should not be used.

Reply:
Resolved in PR #3845.